odl/tomo/geometry/conebeam.py

```diff
@@ -467,11 +473,14 @@ def det_refpoint(self, angle):
             raise ValueError('angle {} is not in the valid range {}.'
                              ''.format(angle, self.motion_params))
 
-        # ASTRA cone_vec geometries
-        return np.array([
-            -self.det_radius * np.sin(angle),
-            self.det_radius * np.cos(angle),
-            self.pitch * angle / (2 * np.pi)])
+        # Distance from 0 to detector
+        offset = self.det_radius * self._source_to_detector
```

Comment:
Maybe it's me, but "offset" confuses me. I read an offset as a constant, once-for-all shift, the zero position everything else starts from. Why not write something like `circle_component` and `pitch_component`? That would anticipate that you intend to add them later.